apps/mobile/app/_layout.tsx

```diff
@@ -61,6 +62,31 @@ function RootLayout() {
                       <Stack.Screen name="notifications" options={{ headerShown: true, title: '' }} />
                       <Stack.Screen name="signers/[address]" options={{ headerShown: true, title: '' }} />
                       <Stack.Screen name="import-signers" options={{ headerShown: true, title: '' }} />
+                      <Stack.Screen
+                        name="import-signers/import-private-key"
+                        options={{ headerShown: true, title: '' }}
+                      />
+                      <Stack.Screen
+                        name="import-signers/import-private-key-success"
+                        options={{
+                          presentation: 'modal',
+                          headerShown: false,
+                        }}
+                      />
+                      <Stack.Screen
+                        name="import-signers/loading-import"
+                        options={{
+                          presentation: 'modal',
+                          headerShown: false,
+                        }}
+                      />
+                      <Stack.Screen
+                        name="import-signers/import-private-key-error"
+                        options={{
+                          presentation: 'modal',
+                          headerShown: false,
+                        }}
+                      />
```

Comment:
Just a nit. 

We'll have quite some typing with those names. 
the folder name is import-signers. Repeating import in the file name afterwards looks redundant. 
import-signers/private-key
import-signers/private-key-success
import-signers/loading
import-signers/private-key-error

convey the same meaning in my opinion.